GitHubRepositories.ps1

```diff
@@ -1066,6 +1072,9 @@ filter Set-GitHubRepository
         By default, rebase-merge pull requests will be allowed.
         Specify this to disallow.
 
+    .PARAMETER AllowAutoMerge
+        Specifies whether to allow auto-merge on pull requests.
```

Comment:
Worthwhile clarifying that this is only valid on public repo's.  Something like this:

```suggestion
        Specifies whether to allow auto-merge on pull requests.  Currently only supported for public repositories.
```

Reply:
Same comment as above.